packages/core/src/queries/setting.test.ts

```diff
@@ -40,22 +40,26 @@ describe('setting query', () => {
   });
 
   it('updateSetting', async () => {
-    const customDomain = 'logto.io';
+    const { adminConsole } = mockSetting;
 
-    const expectSql = sql`
-      update ${table}
-      set ${fields.customDomain}=$1
-      where ${fields.id}=$2
+    /* eslint-disable sql/no-unsafe-query */
+    const expectSql = `
+      update "settings"
+      set
+      "admin_console"=
+      coalesce("admin_console",'{}'::jsonb)|| $1
+      where "id"=$2
       returning *
     `;
+    /* eslint-enable sql/no-unsafe-query */
```

Comment:
using `sql` wrapper to keep consistency? we can replace them later

Reply:
updated